modules/apps/forms-and-workflow/portal-workflow/portal-workflow-api/src/main/java/com/liferay/portal/workflow/constants/WorkflowWebKeys.java

```diff
@@ -48,5 +48,7 @@
 	public static final String WORKFLOW_TAB_INSTANCE = "monitoring";
 
 	public static final String WORKFLOW_TAB_MY_SUBMISSIONS = "my-submissions";
+	
+	public static final String WORKFLOW_NAVIGATION_DISPLAY_CONTEXT = "WORKFLOW_NAVIGATION_DISPLAY_CONTEXT";
```

Comment:
@jeyvison Just for keep the alphabetical order, could you please put the `WORKFLOW_NAVIGATION_DISPLAY_CONTEXT` constant after the `WORKFLOW_DEFINITION_MODIFIED_DATE` one?